File: source/examples/CRUD/BatchUpdate.js
realm.write(() => {
  // Create someone to take care of some dogs.
  const ali = realm.create("Person", { id: 1, name: "Ali" });

  // Find dogs younger than 2.
  const puppies = realm.objects("Dog").filtered("age < 2");

  // Loop through to update.
  puppies.forEach(puppy => {
    // Give all puppies to Ali.
    puppy.owner = ali;
  });
});

File: source/examples/CRUD/BatchUpdate.swift
try! realm.write {
    // Create a person to take care of some dogs.
    let ali = Person(value: ["id": 1, "name": "Ali"])
    realm.add(ali)

    // Find dogs younger than 2.
    let puppies = realm.objects(Dog.self).filter("age < 2")
    
    // Give all puppies to Ali.
    puppies.setValue(ali, forKey: "owner")
}
